tests/Integration/Lti/Builder/Lti1p3MessageBuilderTest.php

```diff
@@ -57,30 +59,34 @@ public function testItCanBuildLtiMessage(): void
         /** @var RegistrationRepository $registrationRepository */
         $registrationRepository = self::$container->get(RegistrationRepository::class);
         /** @var Registration $registration */
-        $registration = $registrationRepository->find('demo');
+        $registration = $registrationRepository->find('testRegistration');
 
         /** @var User $user */
         $user = $this->getRepository(User::class)->find(1);
         $assignment = $user->getLastAssignment();
 
-        $loginHint = new LoginHintDto((string)$user->getUsername(), (int)$assignment->getId());
+        $this->subject
```

Comment:
I believe you do not need this identation.

Reply:
Kept as is for now, we can discuss it with the team